run.py

```diff
@@ -454,7 +455,8 @@ def fit(self):
 	parser.add_argument('-gcn_drop',	dest='dropout', 	default=0.1,  	type=float,	help='Dropout to use in GCN Layer')
 	parser.add_argument('-hid_drop',  	dest='hid_drop', 	default=0.3,  	type=float,	help='Dropout after GCN')
 	parser.add_argument('-disable_gnn_encoder', dest='disable_gnn_encoder', default=False, type=bool, help='Disables the GNN encoder layer')
-
+	parser.add_argument('-enable_early_stopping', dest='enable_early_stopping', default=False, type=bool, help='Enables early stopping with after the amount of epochs that were set.')
```

Comment:
The with in the description is too much imho
```suggestion
	parser.add_argument('-enable_early_stopping', dest='enable_early_stopping', default=False, type=bool, help='Enables early stopping after the amount of epochs that were set.')
```